tox.ini

```diff
@@ -12,7 +12,7 @@ envlist = py, check_codestyle, check_types
 # creates a symlink to the project directory instead of unpacking the sdist.
 usedevelop=true
 
-commands = trial tests
+commands = trial --coverage tests
```

Comment:
This does match up with other projects (i.e. Synapse), however paradoxically we don't actually make use of test coverage stats at the moment. We'd like to, but the output that `trial` creates isn't too useful in its current form. If we enable test coverage in other places such as Sygnal, we'd like to only do so if the output is meaningful.

So for now we'd probably hold back on this change.

Reply:
Yes my bad I did not mean for that change to be added to this PR.